plugin/pkg/auth/authorizer/rbac/bootstrappolicy/controller_policy.go

```diff
@@ -432,6 +432,18 @@ func buildControllerRoles() ([]rbacv1.ClusterRole, []rbacv1.ClusterRoleBinding)
 			eventsRule(),
 		},
 	})
+	if utilfeature.DefaultFeatureGate.Enabled(features.VolumeAttributesClass) {
+		addControllerRole(&controllerRoles, &controllerRoleBindings, rbacv1.ClusterRole{
+			ObjectMeta: metav1.ObjectMeta{Name: saRolePrefix + "vac-protection-controller"},
+			Rules: []rbacv1.PolicyRule{
+				rbacv1helpers.NewRule("list", "watch", "get").Groups(legacyGroup).Resources("persistentvolumeclaims").RuleOrDie(),
+				rbacv1helpers.NewRule("list", "watch", "get").Groups(legacyGroup).Resources("persistentvolumes").RuleOrDie(),
+				rbacv1helpers.NewRule("get", "list", "watch", "update").Groups(storageGroup).Resources("volumeattributesclasses").RuleOrDie(),
```

Comment:
Yep, it follows the same pattern as pvcprotection and pvprotection.

- https://github.com/carlory/kubernetes/blob/kep-3751-finalizer/pkg/controller/volume/pvprotection/pv_protection_controller.go#L176
- https://github.com/carlory/kubernetes/blob/kep-3751-finalizer/pkg/controller/volume/pvcprotection/pvc_protection_controller.go#L206

/cc @msau42 

Should I use a patch instead of an update in this PR?